assets/js/components/KeyMetrics/MetricTileNumeric.js

```diff
@@ -54,7 +54,12 @@ export default function MetricTileNumeric( props ) {
 					{ title }
 				</h3>
 				<div className="googlesitekit-km-widget-tile__body">
-					{ loading && <PreviewBlock width="100%" height="68px" /> }
+					{ loading && (
+						<div className="googlesitekit-km-widget-tile__loading">
+							<PreviewBlock width="100%" height="14px" />
+							<PreviewBlock width="100%" height="53px" />
+						</div>
```

Comment:
Now that I take a closer look (after what I had drafted in the IB), since `div.googlesitekit-km-widget-tile__loading` is duplicated across all three `MetricTile*` components, what do you think about creating a quick little component, such as `MetricTileLoader` for easier potential future updates?

Reply:
I thought about that too, and it makes sense. Will do 👍🏻